main.js

```diff
@@ -15,26 +17,45 @@ const validateUser = (req, res, next) => {
   const accessToken = req.cookies.accessToken;
 
   //decode the cookie to determine what the payload role is 
-  const obj = jwt.decode(accessToken);
+  // const obj = jwt.decode(accessToken);
 
   //NOTE: double check that obj.role accesses the role. consider logging the obj
 
-  const secret = process.env[`ACCESS_TOKEN_${obj.role.toUpperCase()}_SECRET`];
+  const secret = process.env[`ACCESS_TOKEN_${res.locals.role.toUpperCase()}_SECRET`];
 
   //verify token using decoded role. if verification fails, send error
   // result payload comes back as an object with roles and username as keys 
   jwt.verify(accessToken, secret, (err, decoded) => {
     if (err) {
-      throw new Error('Error during role verification in validate user')
+      return next({
+        log: `Express error ${err} during validate user `,
+        status: 400,
+        message: { err: 'INVALID USER' }
+      })
     }
     else {
-      //work with verified decoded payload
       let query = req.body.query;
-      query = query.split(" ");
-      
+      // console.log('Query before split:', query);
+      query = query.replace(/\n/g, "").trim();
```

Comment:
Changing this line so that the newline characters are replaced with spaces, like follows:

query = query.replace(/\n/g, " ").trim();